mollie/Shopware6Composables
# Fix locales code mismatch

fixes #5 

Additional chores:
- fix commands for development in Readme
- update pnpm file to latest format


# Testing
- connect to a working Shopware instance with Mollie plugin installed by providing the credentials in `playground/nuxt.config.ts` 
- remove `locale="en_US"` from components in `playground/app.vue`
- now start the application `pnpm dev` --> The locale of the credit-card form should be the one from Shopware
- now override the locale using for example `locale="it_IT"` as a prop for the components in `app.vue`. Important: always change the locale of all components.
- reload the page, the credit card form should now havev italian labels.
